roles/api-environment/tasks/express-api.yml

```diff
@@ -1,13 +1,22 @@
 ---
   - name: "copy express-api run folder to docker script directory"
     copy:
-      src: "./express-api/"
-      dest: "{{ express_api_docker_path }}"
+      src: "./express-api/run/"
+      dest: "{{ express_api_docker_path }}/run"
       owner: root
       group: root
       mode: "0744"
       force: yes
 
+  - name: "copy pm2 config file"
+    copy:
+      src: "./express-api/{{ pm2_config }}"
+      dest: "{{ pm2_config_path }}"
+      group: "{{ api_group }}"
+      owner: "{{ api_user }}"
+      mode: "0744"
+      force: yes
+
```

Comment:
Can these two tasks combined to one?

Reply:
No, I don't think so since the pm2 config file is going to a different place than the run directory.

Reply:
Right, but I think you can make `src` and `dest` as attributes of a loop dict like [here](https://github.com/osu-mist/ansible-roles/blob/d405932d4a7f95e583e7a02549e1ae3d70cff02e/roles/api-environment/tasks/main.yml#L30-L40)?

Reply:
I could, but `src`, `dest`, `group`, and `owner` would all be different. If only a few properties were different, it might make sense, but I don't think that's the case here. Also, copying the run directory and copying the pm2 config file are fairly different tasks, so it might be good to have them be separate here. For things like the task directly after this one that copies two shell scripts, it makes sense that those two copies are in a loop since the two scripts are related.